src/kvstore/raftex/Host.cpp

```diff
@@ -185,8 +182,8 @@ folly::Future<cpp2::AppendLogResponse> Host::appendLogsInternal(
             {
                 std::lock_guard<std::mutex> g(self->lock_);
                 self->promise_.setValue(r);
+                self->cachingPromise_.setValue(r);
```

Comment:
why the cached request no need retry?

Reply:
Next request will fill the gap.  So we don't need retry here.